tests/resource-events.coffee

```diff
@@ -0,0 +1,56 @@
+###
```

Comment:
would you mind moving these test into the `resource-crud.coffee` file? (create-read-update-delete)